draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -403,7 +403,7 @@ For each claim that is to be selectively disclosed, the Issuer creates a Disclos
 The Issuer MUST create a Disclosure for each selectively disclosable claim as follows:
 
  * Create an array of three elements in this order:
-   1. A salt value. See (#salt-entropy) and (#salt_minlength) for security considerations. The salt value MUST be unique for each claim that is to be selectively disclosed. It is RECOMMENDED to base64url-encode the salt value, producing a string. Any other type that is allowed in JSON MAY be used, e.g., a number.
+   1. A salt value. See (#salt-entropy) and (#salt_minlength) for security considerations. The salt value MUST be unique for each claim that is to be selectively disclosed. It is RECOMMENDED to base64url-encode the salt value, producing a string. Any other type that is allowed in JSON MAY be used, e.g., a number. The salt MUST NOT be disclosed to third parties except the Holder.
```

Comment:
do we need a statement saying that the holder/wallet must protect Disclosures, and in particular, salts with particular care? I mean, the spec of disclosing them is not the only place when salts can get to the hands on an attacker..

Reply:
Being pedantic "MUST NOT be disclosed to third parties except the Holder" suggests that it can't be sent to any Verifier. I do realize this is in the context of the Issuer creating the thing but can see easy confusion.

Reply:
maybe something like this? 

```suggestion
   1. A salt value. See (#salt-entropy) and (#salt_minlength) for security considerations. The salt value MUST be unique for each claim that is to be selectively disclosed. It is RECOMMENDED to base64url-encode the salt value, producing a string. Any other type that is allowed in JSON MAY be used, e.g., a number. The Issuer MUST NOT disclose the salt value to third parties except the Holder.
```